src/internal/base-component/component-metrics.ts

```diff
@@ -10,7 +10,7 @@ export { ComponentConfiguration };
 export function useComponentMetrics(
   componentName: string,
   { packageSource, packageVersion, theme }: PackageSettings,
-  configuration: ComponentConfiguration = { props: {} }
+  { props }: ComponentConfiguration = { props: {} }
```

Comment:
I don't understand why this has changed

Reply:
`configuration` will now include `props` and an optional `analytics` object. We only want to include the `props` as part of the metrics logged.

Reply:
related: https://github.com/cloudscape-design/components/pull/2097#discussion_r1542762845

We are planning to add an extra property for metadata to this object, so it will be even harder to manage all of these

Reply:
Updated. Removed from the ComponentConfiguration. Will align with @pan-kot on their related changes so we both cases work and make sense.